egs/librispeech/ASR/local/prepare_lang_bpe.py

```diff
@@ -204,7 +204,9 @@ def main():
 
     words = word_sym_table.symbols
 
-    excluded = ["<eps>", "!SIL", "<SPOKEN_NOISE>", "<UNK>", "#0", "<s>", "</s>"]
+    excluded = [
+        "<eps>", "!SIL", "!sil", "<SPOKEN_NOISE>", "<UNK>", "<unk>" "#0", "<s>", "</s>"
```

Comment:
```suggestion
        "<eps>", "!SIL", "!sil", "<SPOKEN_NOISE>", "<UNK>", "<unk>", "#0", "<s>", "</s>"
```

Reply:
Ooops... but I checked, it was introduced after I changed from my local `excluded = ["<eps>", "!sil", "<unk>", "#0", "<s>", "</s>"]` to `excluded = ["<eps>", "!SIL", "!sil", "<SPOKEN_NOISE>", "<UNK>", "<unk>" "#0", "<s>", "</s>"]` to preserve the compatibility with Librispeech and by accident forgot the comma, thanks @csukuangfj good catch!